### PR TITLE
Remote Options

### PR DESCRIPTION
**Pull request by sebko**
JIRA URL: https://hoopla.atlassian.net/jira/software/c/projects/CX/boards/30?modal=detail&selectedIssue=CX-128

Preview URLs:
https://developer-docs-git-sebko-featureremote-options-raydiant.vercel.app/docs/inputs/select#remote-options
https://developer-docs-git-sebko-featureremote-options-raydiant.vercel.app/docs/inputs/multi-select#remote-options

I decided not to create a new page/section for this. Instead, I've updated the docs for our select and multi-select to include sections on how to configure remote options.

### CODE REVIEW

File: content/docs/200-inputs/400-select.mdx [hunk 1]
```diff
@@ -7,9 +7,30 @@ Add a _select_ input to your app to allow users to choose from a list of many op
...
+  {
+    "label": "my label",
+    "value": "my-value",
+  },
+  {
+    "label": "my second label",
+    "value": "my-second-value",
+  }
+]
+```
```
Comment (jschr) [inline content/docs/200-inputs/400-select.mdx:33]: Looks great! I think we should document how to set the default value in both the manual and remote sections. 

For remote options I believe you just need to return `{ label: '...', value: '...', default: true }`

Comment (sebko) [reply content/docs/200-inputs/400-select.mdx:33]: Good call! I managed to test this using https://mocki.io/ and it works perfectly! Even with multi-selects.

Updated here: https://github.com/mirainc/developer-docs/pull/46/commits/beb0508d628e520b034cf376caa36d0449e83361

File: content/docs/200-inputs/600-multi-select.mdx [hunk 2]
```diff
@@ -7,7 +7,46 @@ Add a _multi-select_ input to your app to allow users to select multiple values
...
+```json
+[
+  {
+    "label": "my label",
+    "value": "my-value",
+  },
+  {
+    "label": "my second label",
+    "value": "my-second-value",
+    "default": "true"
```
[the remark this answers is not in the document]
Comment (sebko) [reply content/docs/200-inputs/600-multi-select.mdx:44]: Damn, it works! I may as well update the code snippet to set multiple defaults. 
image

File: content/docs/200-inputs/400-select.mdx [hunk 3]
```diff
@@ -7,9 +7,45 @@ Add a _select_ input to your app to allow users to choose from a list of many op
...
+```json
+[
+  {
+    "label": "my label",
+    "value": "my-value",
+  },
+  {
+    "label": "my second label",
+    "value": "my-second-value",
+    "default": "true"
```
[the remark this answers is not in the document]
Comment (sebko) [reply content/docs/200-inputs/400-select.mdx]: Thanks for catching.

Addressed here: https://github.com/mirainc/developer-docs/pull/46/commits/d311946e053b9f5b8be15b6622c5d3e4deb2106b

**Review by jschr — APPROVED** [review]

**Pull request merged**